### ISSUE TITLE
Use appveyor for windows testing

### ISSUE THREAD

**Issue by mrocklin** [dask/dask#371]
http://www.appveyor.com/

It would be nice to have continuous testing of windows as well as the linux support that travis.ci provides.

**Comment by shoyer** [comment]
This is quite easy to setup in my experience for xray:
https://github.com/xray/xray/blob/master/appveyor.yml
https://github.com/xray/xray/blob/master/ci/install_python.ps1

The main thing to note is that Appvveyor's free tier (for open source) is very slow -- expect to wait half an hour for a build to complete. For projects that get lots of PRs (like pandas), the wait can be even more. I think that's why @jreback turned this off.

**Comment by ogrisel** [comment]
It's no longer very slow. New projects are directly run on the new fast appveyor infrastructure (which is as fast as travis). Old projects are progressively been migrated to the new infrastructure. scikit-learn has been mirgrated and we can now monitor pull request prior to merging with appveyor in parallel to travis.

**Comment by ogrisel** [comment]
Here are the details: http://www.appveyor.com/blog/2015/06/23/new-oss-build-environment-and-xamarin-support

**Comment by jcrist** [comment]
We decided to table this for now due to appveyor speed issues. See #383 for reference. Closing.

**Comment by shoyer** [comment]
Hmm. Xray builds with Appveyor seem to have gotten pretty fast, recently. I wonder if this is just a matter of when Appveyor switches you over to the new infrastructure.

**Comment by shoyer** [comment]
This is worth revisiting. In my experience with xarray, Appveyor builds start quite quickly (comparable speed to Travis).

There are [some pernicious bugs](https://groups.google.com/forum/#!topic/xarray/J5uaRsbkqIM) on Windows with big datasets due to numpy using int32 by default for which the only practical way to verify a fix is to run on Windows.

**Comment by pitrou** [comment]
AppVeyor builds now enabled.
